dalle_flow_glid3/sample.py

```diff
@@ -95,29 +98,41 @@ def set_requires_grad(model, value):
 bert.half().eval()
```

Comment:
half-precision mode should only works on CUDA device

Reply:
```python
if device == 'cuda:0':
    bert.half().eval()
else:
    bert.eval()
```
is it good?

I am not sure because original code has this https://github.com/Jack000/glid-3-xl/blob/master/sample.py#L294